### DISCUSSION TITLE
✅ Please focus on performance and listen to the community.

### DISCUSSION THREAD

**Discussion by HBNathan** [elementor/elementor#25615] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

This is less of a feature and more of a plea from myself and the community. Please focus on the performance of the builder as a priority. 

### Describe the solution you'd like.

Some of these new features do nothing but hinder the experience. The general topic on Reddit etc is the decline of Elementor. Now I love Elementor, it was great! But lately, it's slow loading, slow updating, crashes, and buggy (especially nested elements). Please focus on user experience!!

### Additional context

Read the Elementor Sub Reddit! 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @HBNathan, thank you for sharing your feedback! 🙏 

Perhaps you're not familiar with the development cycle of Elementor, but for the past 2 years Elementor has consistently invested and improved on performance. Just take a peak at these articles:

- https://developers.elementor.com/elementor-3-8-developers-update/
- https://developers.elementor.com/elementor-3-9-developers-update/
- https://developers.elementor.com/elementor-3-10-developers-update/
- https://developers.elementor.com/elementor-3-11-developers-update/
- https://developers.elementor.com/elementor-3-12-developers-update/
- https://developers.elementor.com/elementor-3-13-developers-update/
- https://developers.elementor.com/elementor-3-14-developers-update/
- https://developers.elementor.com/elementor-3-15-developers-update/
- https://developers.elementor.com/elementor-3-16-developers-update/
- https://developers.elementor.com/elementor-3-17-developers-update/
- https://developers.elementor.com/elementor-3-18-developers-update/
- https://developers.elementor.com/elementor-3-19-developers-update/
- https://developers.elementor.com/elementor-3-20-developers-update/
- https://developers.elementor.com/elementor-3-21-developers-update/
- https://developers.elementor.com/elementor-3-22-developers-update/
- https://developers.elementor.com/elementor-3-23-developers-update/

All of these releases include performance improvements, most of which are based on user feedback! And we haven't stopped yet. The next release (3.24) will include huge performance improvements in CSS, and we have more plans to continue to improve in this regard.

So, wherever you're getting the information that Elementor is not working on performance, or not giving priority to performance, or even not listening to the community, please, let us know, because we need to show them they are wrong! 💪 

That being said, thank you for the opportunity to show how much work we've been putting on the Editor's performance. 🤓 

I'm closing this request, but will leave the comments opened.

Kind regards
